admin/create-theme/theme-styles.php

```diff
@@ -17,39 +25,37 @@ public static function update_style_css( $style_css, $theme ) {
 		$author        = stripslashes( $theme['author'] );
 		$author_uri    = $theme['author_uri'];
 		$wp_version    = get_bloginfo( 'version' );
+		$wp_min        = $current_theme->get( 'RequiresWP' );
 		$version       = $theme['version'];
 		$requires_php  = $current_theme->get( 'RequiresPHP' );
-		$template      = $current_theme->get( 'Template' );
 		$text_domain   = $theme['slug'];
+		$template      = $current_theme->get( 'Template' ) ? "\n" . 'Template: ' . $current_theme->get( 'Template' ) : '';
+		$license       = $style_data['License'] ? $style_data['License'] : 'GNU General Public License v2 or later';
+		$license_uri   = $style_data['LicenseURI'] ? $style_data['LicenseURI'] : 'http://www.gnu.org/licenses/gpl-2.0.html';
```

Comment:
```suggestion
		$license       = $style_data['License'] ?? 'GNU General Public License v2 or later';
		$license_uri   = $style_data['LicenseURI'] ?? 'http://www.gnu.org/licenses/gpl-2.0.html';
```

Nit

Reply:
That was my first swing but doesn't work because `$style_data['License']` would be `""` if unset, which is not null, so that value (an empty string) is used rather than the backup string provided.